try
  clear a
  a;
  1
catch
end_try_catch
2
